[docs] add docs for disabled services (#4674)
expo/expo: docs/pages/versions/unversioned/guides/adhoc-builds.md

# Troubleshooting

> ## Fixing Disabled Services

Reviewer: apply brent's suggestion

### Push Notifications

> **Note:** Push Notifications are temporarily disabled until we support API tokens for our Push Notification system.

Reviewer: ```suggestion
**Note:** Push Notifications are currently unavailable with ad hoc clients until we complete our work to add an extra authentication layer to the Expo Push Notification service.
```

And comment out the rest of this push notifications guide since it's just confusing to have it.

If you choose not to upload your Push Notifications credentials, Push Notifications will not work on the Expo client. In order to get an Expo client with Push Notifications enabled, you will need to submit another build request and upload or create your push credentials at the prompt.

> We also require you to be logged in order to store your Push credentials.

Reviewer: ```suggestion
We also require you to be logged in order to store your Push Notification credentials.
```

If you are experiencing a different problem with Push Notifications, refer to the [main article](https://docs.expo.io/versions/latest/guides/push-notifications/) for more information.

### Google Maps

> You will need to run `expo client:ios` in a project directory with a valid `app.json`, or pass in the flag to your custom configuration file with `--config <path-to-file.json>`. Make sure you set your API key in `ios.config.googleMapsApiKey` as described [here](https://docs.expo.io/versions/latest/sdk/map-view/#deploying-google-maps-to-a-standalone-app).

Reviewer: ```suggestion
You will need to run `expo client:ios` in a project directory with a valid `app.json`, or pass in the flag to your custom configuration file with `--config <path-to-file.json>`. Make sure you set your Google API key in `ios.config.googleMapsApiKey` as described [here](https://docs.expo.io/versions/latest/sdk/map-view/#deploying-google-maps-to-a-standalone-app).
```
